### PR TITLE
Add Instagram Logo and Link

### PR DESCRIPTION
**Pull request by AnujanM**

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -30,6 +30,16 @@ <h2 class="footer-heading">{{ site.title }}</h2>
               </svg>
             </a>
           </li>{% endif %}
+          
+          {% if site.instagram_username %}<li style="border: #000">
```
Comment (arkon) [inline _includes/footer.html]: You should have the icon inside the circle. This looks weird.

File: _includes/footer.html [hunk 2]
```diff
@@ -30,6 +30,16 @@ <h2 class="footer-heading">{{ site.title }}</h2>
...
...1.121-37.9-5.225-46.778-8.674a78.051,78.051,0,0,1-28.966-18.845,78.053,78.053,0,0,1-18.845-28.966c-3.45-8.877-7.554-22.216-8.674-46.778-1.212-26.564-1.469-34.532-1.469-101.8s0.257-75.233,1.469-101.8c1.121-24.562,5.224-37.9,8.674-46.778A78.052,78.052,0,0,1,78.458,78.458a78.053,78.053,0,0,1,28.966-18.845c8.877-3.45,22.216-7.554,46.778-8.674,26.565-1.212,34.532-1.469,101.8-1.469m0-45.391c-68.418,0-77,.29-103.866,1.516-26.815,1.224-45.127,5.482-61.151,11.71a123.488,123.488,0,0,0-44.62,29.057A123.488,123.488,0,0,0,17.3,90.982C11.077,107.007,6.819,125.319,5.6,152.134,4.369,179,4.079,187.582,4.079,256S4.369,333,5.6,359.866c1.224,26.815,5.482,45.127,11.71,61.151a123.489,123.489,0,0,0,29.057,44.62,123.486,123.486,0,0,0,44.62,29.057c16.025,6.228,34.337,10.486,61.151,11.71,26.87,1.226,35.449,1.516,103.866,1.516s77-.29,103.866-1.516c26.815-1.224,45.127-5.482,61.151-11.71a128.817,128.817,0,0,0,73.677-73.677c6.228-16.025,10.486-34.337,11.71-61.151,1.226-26.87,1.516-35.449,1.516-103.866s-0.29-77-1.516-103.866c-1.224-26.815-5.482-45.127-11.71-61.151a123.486,123.486,0,0,0-29.057-44.62A123.487,123.487,0,0,0,421.018,17.3C404.993,11.077,386.681,6.819,359.866,5.6,333,4.369,324.418,4.079,256,4.079h0Z"/>
```
Comment (arkon) [inline _includes/footer.html]: Prefer just using the `fill` attribute like the other icons.

File: _includes/footer.html [hunk 3]
```diff
@@ -30,6 +30,16 @@ <h2 class="footer-heading">{{ site.title }}</h2>
               </svg>
             </a>
           </li>{% endif %}
+          
+          {% if site.instagram_username %}<li style="border: #000">
+            <a href="https://www.instagram.com/{{ site.instagram_username }}" title="Follow us on Instagram">
+              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
```
Comment (arkon) [inline _includes/footer.html]: You should probably resize/simplify this icon to fit the others. I can do that if you want me to.

**Review by arkon — CHANGES_REQUESTED** [review]

File: _includes/footer.html (hunk 3 above)
Comment (AnujanM) [reply _includes/footer.html]: Yeah My SVG editing skills arn't that great (just grabbed this from Instagram's Site), if you can simplify/resize it that would be highly appreciated. :)

File: _includes/footer.html (hunk 1 above)
Comment (AnujanM) [reply _includes/footer.html]: Yeah I tried with the circle and looked a bit off but if you can make it work @arkon that would be great!

**Comment by ansermino** [comment]
Please remove the twitter link as we are not currently maintaining it.

**Review by ansermino — CHANGES_REQUESTED** [review]
Remove twitter link

File: _includes/footer.html (hunk 3 above)
Comment (arkon) [reply _includes/footer.html]: Done.

**Comment by AnujanM** [comment]
Awesome, thanks a lot @arkon for fixing the logo. Looks great! 👍 🥇

**Pull request merged**